models/pathology_tumor_detection/docs/README.md

```diff
@@ -63,7 +63,8 @@ Inference is performed on WSI in a sliding window manner with specified stride.
 # Model Performance
 
 FROC score is used for evaluating the performance of the model. After inference is done, `evaluate_froc.sh` needs to be run to evaluate FROC score based on predicted probability map (output of inference) and the ground truth tumor masks.
-This model achieve the ~0.92 accuracy on validation patches, and FROC of ~0.72 on the 48 Camelyon testing data that have ground truth annotations available.
+This model achieve the ~0.91 accuracy on validation patches, and FROC of ~0.72 on the 48 Camelyon testing data that have ground truth annotations available.
```

Comment:
Hi @drbeh , as the metric has changed, should the weights be updated as well? May need to change the `large_files.yml`

Reply:
Updated.